api-server/package.json

```diff
@@ -30,11 +32,23 @@
     "request": "^2.88.0",
     "request-promise": "^4.2.4",
     "swagger-ui-dist": "^3.22.1",
-    "winston": "^3.2.1"
+    "copyfiles": "^2.1.0",
```

Comment:
copyfiles should be a devDependency I guess

Reply:
Done